src/engine/specification.md

```diff
@@ -159,6 +159,24 @@ This structure maps on the [`ExecutionPayload`](https://github.com/ethereum/cons
 - `blockHash`: `DATA`, 32 Bytes
 - `transactions`: `Array of DATA` - Array of transaction objects, each object is a byte list (`DATA`) representing `TransactionType || TransactionPayload` or `LegacyTransaction` as defined in [EIP-2718](https://eips.ethereum.org/EIPS/eip-2718)
 
+### WithdrawalV1
+
+This structure maps onto the validator withdrawal object from the beacon chain spec.
+The fields are encoded as follows:
+
+- `index`: `QUANTITY`, 64 Bits
+- `address`: `DATA`, 20 Bytes
+- `amount`: `QUANTITY`, 256 Bits
+
+*Note*: the `amount` value is represented on the beacon chain as a little-endian value in units of Gwei, whereas the `amount` in this structure *MUST* be converted to a big-endian value in units of Wei.
+
+### ExecutionPayloadV2
+
+This structure has the syntax of `ExecutionPayloadV1` and appends a single field: `withdrawals`.
+
+- `ExecutionPayloadV1` fields...
```

Comment:
Even though it's more verbose, I might to prefer to fully copy all the fields for new spec objects. It can help when implementing to avoid jumping through many indirect references.